### DISCUSSION TITLE
How to use Quarto in a Next.js website?

### DISCUSSION THREAD

**Discussion by WillemSleegers** [quarto-dev/quarto-cli#10922] [Q&A]
Hey everyone,

I'm looking into the possibility of creating a blog using [Next.js](https://nextjs.org) and Quarto. I've got a template working in which I have a folder of Quarto documents, which I convert to GitHub flavored markdown format documents, which are then parsed to statically generate the website. 

I realize that this results in the loss of a lot of features that Quarto provides, but I'm curious to see what's possible.

I wonder if this is currently the best approach or whether there are other tricks to creating a blog with Quarto documents that does not rely on the website feature that Quarto provides.

**Comment by mcanouil** [comment]
How do you convert to GitHub Flavoured markdown? `format: gfm`?

**Reply by mcanouil** [reply]
> I wonder if this is currently the best approach or whether there are other tricks to creating a blog with Quarto documents that does not rely on the website feature that Quarto provides.

The only other "alternatives" are Confluence and Hugo.

**Reply by WillemSleegers** [reply]
Yeah, and I've also added `variant: +yaml_metadata_block` to keep the YAML meta data.

**Reply by mcanouil** [reply]
So, there is not much more you can do here.

**Reply by WillemSleegers** [reply]
Unfortunate!

**Reply by cscheid** [reply]
> I realize that this results in the loss of a lot of features that Quarto provides, but I'm curious to see what's possible.

Which features are you looking for?

**Reply by WillemSleegers** [reply]
I suppose the things I want are:
- Create code blocks (works)
- Show output - Tables (haven't tested)
- Show output - Graphs (works if I change the output directory of files and then do some post-processing on the markdown content to fix the paths to the image files)
- Code folding (seems to work)

I think these are the main things. Right now most seems to work but it's a bit hacky; so I'm wondering whether there are any more straightforward solutions.

**Comment by cameronraysmith** [comment]
Related to #10394 and #9292 .
